code/game/gamemodes/roleset/faction/roleset_faction.dm

```diff
@@ -82,6 +82,8 @@
 		if (success_quantity > 1)
 			success_percent = success_quantity / target_quantity
 		cancel(severity, success_percent)
+	
+	return TRUE
```

Comment:
I don't think it should return TRUE if 0 antags were spawned.

Reply:
This is fixed, but I assumed the occurrences strictly meant the event did occur and will never again but I was wrong.